melos.yaml

```diff
@@ -13,6 +13,7 @@ packages:
   - packages/*
   - packages/dynamite/*
   - packages/neon/*
+  - packages/**/example
```

Comment:
Not needed as example packages are automatically included

Reply:
melos will not generate a pubspec_override file if examples are not included.

This means we either need use hard coded paths in the pubspec or use productions releases.

Reply:
Ah I only checked if the pubspec.lock is regenerated.